src/config/tosca.template.js

```diff
@@ -44,6 +44,7 @@ exports.FIELDS = [
   "metadata.sensoroperationalmode",
   "metadata.polarisationmode",
   "metadata.user_tags",
+  "metadata.exists_in_s3",
```

Comment:
this field in Elasticsearch should be vendor agnostic
try `exists_in_object_store` or `rolled_off_object_store`

Reply:
@DustinKLo Updated to exists_in_object_store